tripal_chado/tests/src/Functional/Plugin/NewickImporterTest.php

```diff
@@ -154,20 +154,29 @@ public function testNewickImporterSimpleTest()
     $this->assertGreaterThan(0, $count, "Should have created at least one phylotree but did not.");
 
 
-    // We need to get the type id - polypeptide
-    $results = $chado->query('SELECT * FROM {1:cvterm} WHERE name = :name', [
-      'name' => 'polypeptide'
+    // We need to get the type for 'Phylogenetic tree type', this is used
+    // as the type_id to store the property of the name of the type of tree
+    // (leaf_type for the importer), which for this tree is 'polypeptide'.
+    $results = $chado->query('SELECT * FROM {1:cvterm} T'
+        . ' LEFT JOIN {1:cv} CV ON T.cv_id = CV.cv_id'
+        . ' WHERE T.name = :name AND CV.name = :cvname', [
+      ':name' => 'Phylogenetic tree type',
+      ':cvname' => 'EDAM',
     ]);
-    $type_id = NULL;
+    $bundle_id = NULL;
     foreach ($results as $row) {
-      $type_id = $row->cvterm_id;
+      $bundle_id = $row->cvterm_id;
     }
```

Comment:
We can use getCvtermID() here instead since this is a test.

```suggestion
    // We need to get the type for 'Phylogenetic tree type', this is used
    // as the type_id to store the property of the name of the type of tree
    // (leaf_type for the importer), which for this tree is 'polypeptide'.
    // Vocab: EDAM, Term: Phylogenetic tree type (data:1122).
    $bundle_id = $this->getCvtermID('data','1122');
```

Reply:
Can't commit through the UI, done separately